Users/Install.rst

```diff
@@ -159,7 +159,14 @@ can run
 
 ::
 
-    (venv)$ pip3 install coala-bears --pre
+    (venv)$ git clone https://github.com/coala/coala
+    (venv)$ cd coala/coala
+    (venv)$ pip3 install .
+    (venv)$ cd -
+    (venv)$ git clone https://github.com/coala/coala-bears
+    (venv)$ cd coala/coala-bears
+    (venv)$ pip3 install .
```

Comment:
also the "Installing from source" thing is more for coala devs than users, we might even want to move it to api.coala.io